inc/config.php

```diff
@@ -235,7 +235,7 @@ class='submit'>";
          echo "</div>";
          Html::nullFooter();
       }
-      //exit();
+      exit();
```

Comment:
why?

Reply:
I commented it because I merged with the branch which prevented me from running glpi (update of dev branch is not supported). I was sure I didn't commit it but I was wrong, so I reverted the change